include/boost/dynamic_bitset/dynamic_bitset.hpp

```diff
@@ -128,8 +128,10 @@ class dynamic_bitset
     typedef bool const_reference;
 
     // constructors, etc.
+    dynamic_bitset() : m_bits(Allocator()), m_num_bits(0) {}
```

Comment:
Why `m_bits(Allocator())` instead of `m_bits()`? In fact, this could just be: `dynamic_bitset() : m_num_bits(0) {}`. But no need to invoke `vector(Allocator())` instead of just `vector()` which would already value-initialize the `Allocator`.